low-level/files/opt/seagate/sspl/setup/setup_logger.py

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2001-2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License along
+# with this program. If not, see <https://www.gnu.org/licenses/>. For any questions
+# about this software or licensing, please email opensource@seagate.com or
+# cortx-questions@seagate.com.
+
+"""
+ ****************************************************************************
+  Description:       logging utility for the sspl_setup
+ ****************************************************************************
+"""
+
+import sys
+import logging
+
+logger_facility = "sspl-setup"
+_logger = logging.getLogger(logger_facility)
+
+def init_logging(syslog_host="localhost", syslog_port=514):
+    """Initialize logging to log for sspl-setup"""
```

Comment:
Can we keep this logger module under framework.?
Let this be a class. And __init__() to take logger_fecility variable.
So that reusability will be higher. Support bundle logger also can use this instead of keeping similar code there.

Reply:
@mariyappanp I tried it earlier too. but manifest & support bundle scripts logs at different paths and executes independently, and as per my understanding, SysLogHandler doesn't listen to that socket as port 514 is binded already.

Reply:
Okay, then we can ignore it. 
SSPL service is also having logging module.
Is that the case for sspl_setup and sspl service as well?

Reply:
yes.

Reply:
Okay.